src/main/java/com/futurecollars/Triangle.java

```diff
@@ -0,0 +1,21 @@
+package com.futurecollars;
+
+public class Triangle {
+    public class Triangle {
```

Comment:
??? jak już kopiujesz to chociaż zacieraj ślady :)

Reply:
:P pisałem w brudnopisie :P na konsultacjach ze znajomym